app/models/user.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
+include UnicodeUtils
```

Comment:
I see, so it's not about it 'being Unicode' so much as this being an app internationalized in English.